Libraries/MAXUSB/include/core/usb.h

```diff
@@ -135,6 +136,48 @@ typedef struct {
  */
 int MXC_USB_Init(maxusb_cfg_options_t *options);
 
+#ifdef MAX32690
+// Clock setter mux APIs are currently only supported on the MAX32690.  It does not exist
+// on earlier hardware revisions such as the MAX32650
+
+#ifndef mxc_usb_clock_t
+#warning "mxc_usb_clock_t" not implemented.  See note in usb.h on "MXC_USB_SetClockSource"
+typedef enum {
+  MXC_USB_CLOCK_0 = 0,
+  MXC_USB_CLOCK_1 = 1,
+  MXC_USB_CLOCK_2 = 2
+} mxc_usb_clock_t;
+#endif
+
+/**
+ * @brief Set the input clock source to the USB peripheral.
+ *
+ * @param   clock_source Input clock source
+ * @note    (Developers): "mxc_usb_clock_t" should be defined as a macro in the top-level "max32xxx.h" file
+ *          so that the pre-processor can check for its existence.  Ex:
```

Comment:
Can you add this note to the header device files (**max32690.h**)? Some developers only pull the CMSIS.

Reply:
https://github.com/analogdevicesinc/msdk/pull/1108/commits/5de0feaaeb656af33f775f8aa6c99d2687a1d0fb